INSTALL.md

```diff
@@ -375,9 +375,12 @@ To configure Synapse to expose an HTTPS port, you will need to edit
 * You will also need to uncomment the `tls_certificate_path` and
   `tls_private_key_path` lines under the `TLS` section. You can either
   point these settings at an existing certificate and key, or you can
-  enable Synapse's built-in ACME (Let's Encrypt) support.  Instructions
+  enable Synapse's built-in ACME (Let's Encrypt) support. Instructions
   for having Synapse automatically provision and renew federation
-  certificates through ACME can be found at [ACME.md](docs/ACME.md).
+  certificates through ACME can be found at [ACME.md](docs/ACME.md). If you
+  are using your own certificate and key, be sure to use a certificate that
```

Comment:
wait, this is misworded. A certificate can't include a chain: you need the full chain *of* certificates.

"use a `.pem` file which includes the full certificate chain including any intermediate certificates".

Likewise for the config text,